src/applications/mhv/medical-records/reducers/careSummariesAndNotes.js

```diff
@@ -77,7 +85,8 @@ const convertProgressNote = record => {
     name: extractName(record),
     type: extractType(record),
     dateSigned: record.date ? formatDateLong(record.date) : EMPTY_FIELD,
-    signedBy: extractAuthor(record) || EMPTY_FIELD,
+    signedBy: extractAuthenticator(record) || EMPTY_FIELD,
+    coSignedBy: extractAuthor(record) || EMPTY_FIELD,
```

Comment:
Double check this. I think the mapping is:
- signedBy: DocumentReference.author.reference (from NoteTO.author)
- cosignedBy: DocumentReference.authenticator.reference (from NoteTO. approvedBy)

Here is the logic in existing MHV:
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/33fc5b7d-2e8b-4770-b994-29f1964efc03)

Here is [John's mapping](https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition-VA.MHV.PHR.note-mappings.html#mappings-for-via-to-mhv-fhir-phr-noteto).